src/plugin.scss

```diff
@@ -84,38 +85,24 @@
     overflow: auto;
     position: absolute;
     padding-bottom: 15px;
-    top: 75px;
+    top: 0;
     left: 15px;
     right: 15px;
     bottom: 61px; /* dialog padding + ok button + 10 */
 }
 
 .vjs-errors-headline {
-    font-size: 22px;
-    font-weight: bold;
-    margin-top: 0;
-}
-
-.vjs-errors-close-button {
-    background-color: transparent;
     font-size: 14px;
     font-weight: bold;
-    cursor: pointer;
-    float: right;
-    margin: 5px;
-    border: 0;
-    color: #999;
+    padding-right: 3em;
 }
 
-.vjs-errors-close-button:before {
-    content: 'X';
-}
-
-.vjs-errors-close-button:hover {
-    color: #FFF;
+.vjs-close-button.vjs-control.vjs-button {
```

Comment:
this is very generic and will apply to all close buttons and not just the error's close button.